README.md

```diff
@@ -35,7 +35,7 @@ Nethogs depends on `ncurses` for the text-based interface and `libpcap` for user
 
 #### Yum-based distro's
 
-    yum install gcc-c++ libpcap-devel.x86_64 libpcap.x86_64 ncurses*
+    yum install gcc-c++ libpcap-devel.x86_64 libpcap.x86_64 ncurses\*
```

Comment:
This would fail if there are any files in the current directory that start with "ncurses". Perhaps quotes would be safer (tested with bash and zsh):
```suggestion
    yum install gcc-c++ libpcap-devel.x86_64 libpcap.x86_64 "ncurses*"
```